src/hooks/useDittoSingleText.ts

```diff
@@ -1,27 +1,31 @@
 import { useContext } from "react";
-import { DittoContext, SourceDetector } from "../lib/context";
-import { nullError } from "../lib/utils";
+import { Plurals } from "../components/Ditto";
+import { DittoContext, SourceDetector, Variables, Count } from "../lib/context";
+import { nullError, interpolateVariableText } from "../lib/utils";
 
 interface useDittoSingleTextProps {
   projectId?: string;
   textId: string;
+  variables: Variables;
+  count?: Count;
 }
 
 export const useDittoSingleText = (props: useDittoSingleTextProps) => {
-  const { projectId, textId } = props;
+  const { projectId, textId, variables = {}, count } = props;
   const { source, variant, options } = useContext(DittoContext);
 
   if (!projectId) return nullError("No Project ID provided.");
 
   if (variant) {
     const data = source?.[projectId]?.[variant];
     if (data) {
+
       if (SourceDetector.isStructured(data)) {
-        return data[textId].text;
+        return interpolateVariableText(data[textId], variables).text;
```

Comment:
I think the `count` argument is supposed to be passed to all of the `interpolateVariableText` calls in this file, right?
![image](https://user-images.githubusercontent.com/19500384/149528794-971b25ab-e8a4-40fe-be4d-f6be2e678250.png)

Reply:
I pushed a small commit for this to get the app to compile for testing